go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/AdaLogics/go-fuzz-headers v0.0.0-20210715213245-6c3934b029d8
 	github.com/Microsoft/go-winio v0.5.1
-	github.com/Microsoft/hcsshim v0.9.3
+	github.com/Microsoft/hcsshim v0.9.4
```

Comment:
It looks like `main` is on an older version than the release branches now; which means that the containerd 1.7 will (in the current state) introduce regressions in this area 
https://github.com/containerd/containerd/blob/14559264164276b428f076091ca867ef982ceeff/go.mod#L8


could you open a pull request against `main` to;

- for-ward port https://github.com/containerd/containerd/pull/6968 (which also was in the 1.6 branch only, but no corresponding PR for `main`)
- and a commit to update the version to v0.9.4 ?

After that's merged, we can do a _backport_  / cherry-pick into the release branch(es))

Reply:
@thaJeztah This was "semi"-intentional for now as we were looking to move main to a 0.10.0-beta tag very shortly. v0.9.2 doesn't even have the code that introduced the regression so we should be fine in that regard. If we want to do the incremental move main to 0.9.4 first in main as well I'm fine with that, let me know

Reply:
Merged 0.9.4 to main, resolving